giraf/convert.go

```diff
@@ -131,3 +130,54 @@ func fromStringToQual(s string) []uint8 {
 	}
 	return answer
 }
+
+func ToString(g *Giraf) string {
+	buf := &strings.Builder{}
+	buf.Grow(400)
+	var err error
+	_, err = buf.WriteString(g.QName)
+	common.ExitIfError(err)
+	err = buf.WriteByte('\t')
+	common.ExitIfError(err)
+	_, err = buf.WriteString(strconv.Itoa(g.QStart))
```

Comment:
Is the style of processing these values one-at-a-time chosen because fmt.Sprintf is slower?